sql/firefox_desktop_exact_mau28_v1.sql

```diff
@@ -1,12 +1,14 @@
+CREATE TABLE
+  firefox_desktop_exact_mau28_v1 AS
 SELECT
   submission_date,
-  CURRENT_DATETIME() AS generated_time,
   SUM(mau) AS mau,
   SUM(wau) AS wau,
-  SUM(dau) AS dau
+  SUM(dau) AS dau,
+  SUM(visited_5_uri_mau) AS visited_5_uri_mau,
+  SUM(visited_5_uri_wau) AS visited_5_uri_wau,
+  SUM(visited_5_uri_dau) AS visited_5_uri_dau
 FROM
-  firefox_desktop_exact_mau28_by_dimensions_v1
-WHERE
-  submission_date = @submission_date
+  `moz-fx-data-derived-datasets.telemetry.firefox_desktop_exact_mau28_by_dimensions_v1`
```

Comment:
If this isn't going to be a view, then this probably doesn't have to be fully specified.

Reply:
it's going to be a view